delivery_correos_express/tests/test_correos_express_request.py

```diff
@@ -65,37 +66,37 @@ def test_03_send_api_request_error(self, mock_post):
                 "POST", "https://test.url", data={"test": "data"}
             )
 
-    def test_04_check_for_error_shipment_success(self):
+    def test_03_check_for_error_shipment_success(self):
```

Comment:
No need to change all the method names, as this diff is not adding value. No matter if there's a hole. At the end, they are going to be run in the same order.